auth/src/main/java/org/aerogear/mobile/auth/authenticator/oidc/OIDCAuthenticatorImpl.java

```diff
@@ -60,12 +63,14 @@
      * @param authStateManager {@link AuthStateManager}
      * @param authorizationServiceFactory {@link AuthorizationServiceFactory}
      * @param jwksManager {@link JwksManager}
+     * @param httpModule {@link HttpServiceModule} Module used to make HTTP requests for
+     *        authenticator.
      */
     public OIDCAuthenticatorImpl(final ServiceConfiguration serviceConfiguration,
                     final AuthServiceConfiguration authServiceConfiguration,
                     final AuthStateManager authStateManager,
                     final AuthorizationServiceFactory authorizationServiceFactory,
-                    final JwksManager jwksManager) {
+                    final JwksManager jwksManager, final HttpServiceModule httpModule) {
```

Comment:
Just to make note, as it was mentioned in the other PR, with 6 param's it might be an idea to look at adding a builder to this class as it seems to be doing a lot of the heavy lifting right now. Out of the scope of this PR, but worth nothing all the same.